docker-compose.test.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  prod:
+    build: .
+    command: pnpm run start
+    restart: unless-stopped
+    volumes:
+      - .:/app
+    ports:
+      - '3001:3001'
+    networks:
```

Comment:
It will be better to remove this testing network
You don’t need it if you use ports, because you expose 3001 to your localhost